.github/workflows/deploy.yml

```diff
@@ -332,6 +332,13 @@ jobs:
                   aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
                   aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
                   aws-region: us-east-2
+
+            - name: Login to Docker Hub
```

Comment:
sorry, will separate it. this is to avoid some CI failures I noticed when the docker API rate limits us (using our paid logged in account avoids the rate limit)